### ISSUE TITLE
Sanitize removes negative value in style

### ISSUE THREAD

**Issue by siddhartham** [rails/rails-html-sanitizer#37]
Sanitize removes a style like "margin:-4px".
This test fails
```ruby
input = '<p style="color: #000; margin: -9px;"></p>'
assert_equal input, white_list_sanitize(input, attributes: %w(style))
```

**Comment by siddhartham** [comment]
The problem is in loofah. added a fix there and sent a pull request
https://github.com/flavorjones/loofah/pull/85

**Comment by kaspth** [comment]
Nice find and thanks for working on this, :+1:.

**Comment by kaspth** [comment]
Fixed in Loofah v2.0.2 via flavorjones/loofah#85
